e2e/scenario/scenario.go

```diff
@@ -77,6 +77,10 @@ func (t *Template) scenarios() []*Scenario {
 		t.marinerv2AirGap(),
 		t.azurelinuxv2AirGap(),
 		t.marinerv2ARM64AirGap(),
+		t.ubuntu2204SystemdChronyDropin(),
+		t.ubuntu2204ChronyRestarts(),
```

Comment:
@anujmaheshwari1  will we be duplicating ALL of the 2204 e2e scenarios with 2404 ? 

Should we look at improving those scenarios to be OS agnostic, looping them across all OS family /version

Reply:
I was wondering this too. It should be feasible to say a scenario runs on several VHDs rather than specifying only one. On saying that, the test for chrony on Ubuntu1804 is slightly different to 2204. It wouldn't surprise me if there were differences for Ubuntu2404 as well.

Reply:
we have some scenarios which are responsible for only getting coverage on the particular SKU, rather than testing a particular scenario, though I'd be open to the idea of creating OS-agnostic scenarios

Reply:
Resolving convo as it's a really a feature add for a different PR.